cypress/plugins/index.js

```diff
@@ -18,6 +18,16 @@ module.exports = (on, config) => {
 
   require('@cypress/code-coverage/task')(on, config);
   on('file:preprocessor', require('@cypress/code-coverage/use-babelrc'));
+  on('task', {
+    log(message) {
+      console.log(message);
+      return null;
+    },
+    table(message) {
+      console.table(message);
+      return null;
+    }
+  });
```

Comment:
I was pretty confused by why these were `on('task')` instead of registered as custom commands, like `cy.consoleLog` or `cy.consoleTable`, until I read up on the docs for `cy.task`. It would be great to include that extra knowledge and context here for devs less familiar with Cypress.
```suggestion

  // `cy.task` executes code in Node and allows us to output to the terminal.
  // @see https://docs.cypress.io/api/commands/task
  on('task', {
    log(message) {
      console.log(message);
      return null;
    },
    table(message) {
      console.table(message);
      return null;
    }
  });
```